roles/ceph-osd/tasks/activate_osds.yml

```diff
@@ -1,4 +1,12 @@
 ---
+- name: enable osd service instance(s) (for or after infernalis)
+  command:  "ln -s /usr/lib/systemd/system/ceph-osd@.service /etc/systemd/system/multi-user.target.wants/ceph-osd@{{ item.0 }}.service"
```

Comment:
can you please use the 'file' module to create your symlink?
http://docs.ansible.com/ansible/file_module.html